src/app/backend/resource/dataselect/dataselect.go

```diff
@@ -122,7 +124,7 @@ func (self *DataSelector) Filter() *DataSelector {
 				matches = false
 				continue
 			}
-			if filterBy.Value.Compare(v) != 0 {
+			if !filterBy.Value.StartsWith(v) {
```

Comment:
I don't think so. For example contains would return true in case of object `heapster` and search string such as `ster`. I suppose we do not want that.